webapp/src/containers/modals/SubscribeModal/filters/boards.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {useCallback, useEffect, useMemo} from 'react';
+
+import pluginConstants from 'pluginConstants';
+import {filterLabelValuePairAll} from 'pluginConstants/common';
+
+import Dropdown from 'components/dropdown';
+
+import usePluginApi from 'hooks/usePluginApi';
+
+import {formLabelValuePairs} from 'utils';
+
+type BoardsFilterProps = {
+    organization: string
+    projectId: string
+    eventType: string
+    selectedAreaPath: string
+    handleSelectAreaPath: (value: string, name?: string) => void
+    setIsFiltersError: (value: boolean) => void
+}
+
+const BoardsFilter = ({
+    organization,
+    projectId,
+    eventType,
+    selectedAreaPath,
+    handleSelectAreaPath,
+    setIsFiltersError,
+}: BoardsFilterProps) => {
+    const {subscriptionFiltersNameForBoards, subscriptionFiltersForBoards} = pluginConstants.form;
+
+    const {
+        getApiState,
+        makeApiRequestWithCompletionStatus,
+    } = usePluginApi();
+
+    const getSubscriptionFiltersRequest = useMemo<GetSubscriptionFiltersRequest>(() => ({
```

Comment:
I don't thinkg using `useMemo` here gives us an advantage. What do you think?

Reply:
`selectedAreaPath` is a prop here which when changed in the parent this entire component will be re-rendered but we don't need that so, I think we should keep it